components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -71,12 +71,12 @@ enum SuggestionGenerationStatus {
 struct SiteInfo {
   // Title is present if the site has associated page content
   string? title;
-  // the content so long that it's being truncated
-  bool is_content_truncated;
```

Comment:
I'm surprised this isn't breaking iOS, but CI is passing so I suppose we're ok cc @Brandon-T